packages/cli/src/cmds/index/rpcServer.ts

```diff
@@ -45,10 +51,14 @@ export default class RPCServer {
     log(`Available JSON-RPC methods: ${Object.keys(rpcMethods).sort().join(', ')}`);
     warn(`Consult @appland/rpc for request and response data types.`);
 
-    this.server = new jayson.Server(rpcMethods).http();
-    this.server.listen(this.bindPort).on('listening', () => {
-      assert(this.server);
-      const address = this.server.address();
+    const server = new jayson.Server(rpcMethods);
```

Comment:
It's worth mentioning that the server won't be instantiated until all AppMaps are polled to determine whether or not they need to be indexed. On a local project of mine, it took about 80 seconds before the server was started.

Reply:
How many AppMaps was that? It should just be checking the timestamp of the AppMap JSON file against the timestamp in the index.

Reply:
Looks like I have about 7000 :smile:

Reply:
It should only have to check the timestamp of each AppMap and then read one small file, I wouldn't expect that to take 80 seconds for 7,000 files.